yt_dlp/update.py

```diff
@@ -21,7 +21,13 @@
     traverse_obj,
     version_tuple,
 )
-from .version import UPDATE_HINT, VARIANT, __version__
+from .version import (
+    CHANNEL,
+    RELEASE_GIT_HEAD,
+    UPDATE_HINT,
+    VARIANT,
+    __version__,
+)
 
 REPOSITORY = 'yt-dlp/yt-dlp'
```

Comment:
Can be tested by changing this to
```suggestion
REPOSITORY = 'pukkandan/yt-dlp-dev'
```